backend/battles/templates/battles/battle_form.html

```diff
@@ -13,20 +13,44 @@ <h1>Create a new Pokebattle</h1>
     <span class="field-error">{{ form.opponent.errors }}</span>
   </div>
   <div class="field-wrapper">
-    <label for="{{ form.creator_pokemon_1_input.id_for_label }}">Pokemon 1:</label>
-    {{ form.creator_pokemon_1_input }}
-    <span class="field-error">{{ form.creator_pokemon_1_input.errors }}</span>
+    <label for="{{ form.creator_pokemon_1.id_for_label }}">Pokemon 1:</label>
+    {{ form.creator_pokemon_1 }}
+    <span class="field-error">{{ form.creator_pokemon_1.errors }}</span>
   </div>
   <div class="field-wrapper">
-    <label for="{{ form.creator_pokemon_2_input.id_for_label }}">Pokemon 2:</label>
-    {{ form.creator_pokemon_2_input }}
-    <span class="field-error">{{ form.creator_pokemon_2_input.errors }}</span>
+    <label for="{{ form.creator_pokemon_2.id_for_label }}">Pokemon 2:</label>
+    {{ form.creator_pokemon_2 }}
+    <span class="field-error">{{ form.creator_pokemon_2.errors }}</span>
   </div>
   <div class="field-wrapper">
-    <label for="{{ form.creator_pokemon_3_input.id_for_label }}">Pokemon 3:</label>
-    {{ form.creator_pokemon_3_input }}
-    <span class="field-error">{{ form.creator_pokemon_3_input.errors }}</span>
+    <label for="{{ form.creator_pokemon_3.id_for_label }}">Pokemon 3:</label>
+    {{ form.creator_pokemon_3 }}
+    <span class="field-error">{{ form.creator_pokemon_3.errors }}</span>
   </div>
   <button type="submit">Create Battle</button>
 </form>
 <a class="nav-link" href="{% url 'battles:battle_list' %}">Back to list</a>
+
+<script type="text/javascript" src="{% static 'admin/js/vendor/jquery/jquery.js' %}"></script>
+
+{{ form.media }}
+<script>
+  (function($) {
```

Comment:
What is this doing? Why do you need it?

Reply:
Added a code comment.

Reply:
`django-autocomplete-light configuration`? I looked the docs and couldn't find where it specified we need this can you point it to me? Also, perhaps it's a good idea to link this docs here in the code

Reply:
You're right. I improved the comment explaining what it does and added the link to the lib documentation.